src/containers/SignUp/SignUp.js

```diff
@@ -12,36 +12,32 @@ class SignUp extends Component {
 
     state = {
         didSignUp: false,
-        state: null,
         district: null,
         communicationMethods: null,
         postCallerDetailsError: null
     }
 
     handleFormSubmit = (fieldsValues) => {
-        const {congressionalDistrict} = {...fieldsValues};
-        const state = congressionalDistrict[0];
-        const district = congressionalDistrict[1];
-        
+        const {district, zipCode, firstName, lastName, phone, email, } = {...fieldsValues};
         const caller = {
-            districtId: fieldsValues.districtId,
-            zipCode: fieldsValues.zipCode,
-            firstName: fieldsValues.firstName,
-            lastName: fieldsValues.lastName,
+            districtId: district.districtId,
+            zipCode: zipCode,
+            firstName: firstName,
+            lastName: lastName,
```

Comment:
Small nit thing, but here's some sugar you can use if the key and variable name are the same:

```suggestion
            districtId: district.districtId,
            zipCode,
            firstName,
            lastName,
```